docs/javascripts/maatregelen.js

```diff
@@ -0,0 +1,219 @@
+let maatregelen = [];
+
+// Function to load maatregelen from localStorage
+function loadMaatregelenFromLocalStorage() {
+    const storedMaatregelen = localStorage.getItem('maatregelen');
+    if (storedMaatregelen) {
+        maatregelen = JSON.parse(storedMaatregelen);
+        renderMaatregelenList();
+        renderStoredMaatregelen();
+        updateDisplay();
+    }
+}
+
+// Function to construct the correct path to the JSON file
+function getFilePath() {
+    return "../../maatregelen.json";
```

Comment:
Would it not be better to use the absolute path? then this script will work on all pages.